### DISCUSSION TITLE
How to update 'slug' during update?

### DISCUSSION THREAD

**Discussion by gundogduyakici** [spatie/laravel-sluggable#253] [General]
Hello,

Any problem if you add the post but I also want to have it when I update. (for some pages). How can we make this situation? What needs to be done in the model file? Is there a method for this? Below are the codes in my model file;

```
public function getSlugOptions() : SlugOptions
{
    return SlugOptions::create()
        ->generateSlugsFrom('name')
        ->saveSlugsTo('slug')
        ->usingLanguage('de');
}
```

**Comment by Horlerdipo** [comment]
Hello @gundogduyakici , did you find a solution to this

**Reply by Horlerdipo** [reply]
UPDATE:
I used $model->generateSlug() to generate the slug and then did the update
Check it out here 
https://github.com/spatie/laravel-sluggable#regenerating-slugs
